test/transport.test.js

```diff
@@ -366,3 +368,15 @@ test('pino.transport with target #pino/pretty', async ({ match, teardown }) => {
   const actual = await readFile(destination, 'utf8')
   match(strip(actual), /\[.*\] INFO.*hello/)
 })
+
+test('stdout in worker', async ({ not }) => {
+  let actual = ''
+  const child = execa(process.argv[0], [join(__dirname, 'fixtures', 'transport-main.js')])
```

Comment:
this looks more like a script than a fixture, no?

Reply:
We place those in `fixtures` - they are almost all the same.